libfreerdp/core/gateway/rdg.c

```diff
@@ -1583,6 +1575,18 @@ static long rdg_bio_ctrl(BIO* bio, int cmd, long arg1, void* arg2)
 		else
 			status = 1;
 	}
+	else if (cmd == BIO_C_GET_EVENT || cmd == BIO_C_GET_FD)
+	{
+		/*
+		 * A note about BIO_C_GET_FD:
```

Comment:
This equally applies to BIO_C_GET_EVENT, i.e. there are two events and only OUT connection event is returned.
Not sure, though, if such a comment is even necessary here, IMHO just mentioning the issue number in commit message is enough.